test/integration/spec/logger.js

```diff
@@ -124,6 +124,32 @@ describe('logger', function() {
     assert.equal(room.name, loggedRoomName);
   });
 
+  it('should log signaling events', async () => {
+    room = await connect(token, Object.assign({ name: sid }, defaults));
+    const callSpies = loggerCb.getCalls();
+    assert(!!callSpies.length);
+
+    // ensure that signaling events were fired for early/connecting/open events.
+    let early = false;
+    let connecting = false;
+    let open = false;
+    callSpies.forEach(callSpy => {
+      const { message, data } = callSpy.args[0];
+      if (message === 'event' && data.group === 'signaling') {
+        assert(typeof data.elapsedTime === 'number');
+        assert(typeof data.timestamp === 'number');
+        assert(typeof data.level === 'string');
+        assert(typeof data.name === 'string');
+        early = early || data.name === 'early';
+        connecting = connecting || data.name === 'connecting';
+        open = open || data.name === 'open';
```

Comment:
Should we also test for `closed`? Also, `waiting` was mentioned in the change log. Should we also test for `waiting`?

Reply:
we cant test for closed since it happens only after we disconnect from room. and `waiting` is not a reproducible state.

Reply:
I will update to check the call spies after disconnect. This will let us check for 'closed' event as well.